### ISSUE TITLE
[ALM Accelerator - BUG] GetAzureTenantInformation flow failed.

### ISSUE THREAD

**Issue by RavichandraPulipati** [microsoft/coe-starter-kit#7526]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Hi Team, 

We have tried configuring ALM Accelerator in an environment by following a Microsoft document available in the link, https://learn.microsoft.com/en-us/power-platform/guidance/alm-accelerator/setup-admin-tasks. As per the document we have downloaded a managed solution from https://github.com/microsoft/coe-starter-kit/releases and imported to and noticed that a flow is getting failed. Details are available here, could you please help.

Flow Name: GetAzureTenantInformation
Step/Action: Call Graph to get the tenant information
Error: Resource not found for the segment 'v1.0'.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

1.0.20231109.3

### What app or flow are you having the issue with?

Solution Name: CenterofExcellenceALMAccelerator, Flow Name: GetAzureTenantInformation

### What method are you using to get inventory and telemetry?

None

AB#2121

**Comment by RajeevPentyala** [comment]
@RavichandraPulipati Could you please verify if the user authenticating the Graph API call has the necessary access?
![image](https://github.com/microsoft/coe-starter-kit/assets/104883923/efe19d45-c817-4c71-8147-d72e3ac46746)

You can utilize [Graph Explorer](https://developer.microsoft.com/en-us/graph/graph-explorer) as illustrated below, to test user access.
![image](https://github.com/microsoft/coe-starter-kit/assets/104883923/95e64fc2-a9d5-4aec-9fbb-a7c487ddaf26)

**Comment by RavichandraPulipati** [comment]
Hi,

Thanks for your prompt response.

I cross-checked the privileges and everything appears good! Please refer to the screenshots below.

***@***.***


***@***.***
Best Regards,

Narendra Reddy
Dynamics CRM Technical

Trelleborg Industrial Solutions - Digital Solutions
Customer Experience Competence Centre
Mobile: +91 8142112298
Microsoft ***@***.***>
***@***.******@***.***>
www.trelleborg.com<http://www.trelleborg.com/>
Follow us on:
LinkedIn<https://www.linkedin.com/company/trelleborg-sealing-profiles?trk=company_logo> Facebook<https://www.facebook.com/Trelleborggroup/> Twitter<https://twitter.com/trelleborggroup> YouTube<https://www.youtube.com/trelleborggroup>

From: Ravichandra Pulipati ***@***.***>
Sent: Thursday, January 25, 2024 4:40 PM
To: microsoft/coe-starter-kit ***@***.***>; microsoft/coe-starter-kit ***@***.***>; Reddy Narendra ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [microsoft/coe-starter-kit] [ALM Accelerator - BUG] GetAzureTenantInformation flow failed. (Issue #7526)

++ @Reddy ***@***.***>



Internal - General Use

**Comment by Roelof-BAM** [comment]
The issue is that the flow GetAzureTenantInformation expects in the action ParseJson the field State to have a value. 
```
          "state": {
            "type": "string"
          },
```
When we filled that field in the tenant settings with a value, we did not have the error anymore.
But it would be beter to change the flow so that null values for State are allowed. So change the Json schema for field  to 
`"state": null,`
